Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/core/impl/ConceptManager.java

```diff
@@ -572,4 +582,11 @@ private int getConceptsCountForConceptList(String listName) {
         alternativeIdService.addAlternativeIds(conceptEntries);
         return conceptEntries;
     }
+
+    @Override
+    public void addWordnetConceptEntry(ConceptEntry entry, String userName)
```

Comment:
Yes, I thought about that, but the method addConceptListEntry(ConceptEntry entry, String userName) has an if condition that checks whether concept list is "wordnet", if so it throws DictionaryModificationException. Maybe I could reuse this method if I remove this if condition. Let me know if I can remove this and reuse this method? Will update the test cases as well.